copier.yaml

```diff
@@ -113,3 +129,52 @@ default_py:
       value: '3.11'
   help: 'Choose the default Python version for development, documentation generation, and package build:'
   type: str
+
+copyright_license:
+  type: str
+  help: Your project's license. If you have no idea how to choose one, please see https://choosealicense.com/.
```

Comment:
Suggest to use the help message from ChatGPT. :-)

![image](https://github.com/serious-scaffold/serious-scaffold-python/assets/726061/a844b608-8459-458c-bf5b-e7dcd8c96c1f)

Text version:
```
help: 'Choose the project's license (e.g., "MIT", "GPL-3.0").'

help: 'Enter the copyright year or range (e.g., "2020-2023").'

help: 'Name(s) or organization(s) holding the copyright.'
```